coil-base/src/main/java/coil/annotation/ExperimentalCoil.kt

```diff
@@ -0,0 +1,10 @@
+package coil.annotation
+
+/**
+ * Marks declarations that are still **experimental**.
+ * Targets marked by this annotation design is incubating and may contain breaking changes in the future.
+ */
+@MustBeDocumented
+@Retention(AnnotationRetention.BINARY)
+@Experimental(Experimental.Level.WARNING)
+annotation class ExperimentalCoil
```

Comment:
Adding this for future use.

Reply:
I lied :). Adding this to the transitions API for one release in case it needs to be modified to support other use cases.